jme3-core/src/main/java/com/jme3/anim/tween/action/BaseAction.java

```diff
@@ -6,10 +6,35 @@
 
 import java.util.List;
 
+/**
+ * A Concrete fast Implementation for the interface #{@link Action},
```

Comment:
This implementation is fast compared to what?
Also, "Concrete" and "Implementation" should not be capitalized.

Reply:
Better. I would say, "A simple implementation of the Action interface."